examples/secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request/1-Prepare-Request-200_OK.json

```diff
@@ -39,179 +21,197 @@
             }
           }
         ],
+        "eventCoding": {
+          "code": "prescription-order",
+          "display": "Prescription Order",
+          "system": "https://fhir.nhs.uk/CodeSystem/message-event"
+        },
         "focus": [
           {
             "reference": "urn:uuid:56166769-c1c4-4d07-afa8-132b5dfca666"
           },
           {
             "reference": "urn:uuid:a54219b8-f741-4c47-b662-e4f8dfa49ab6"
           }
-        ]
+        ],
+        "id": "3599c0e9-9292-413e-9270-9a1ef1ead99c",
+        "resourceType": "MessageHeader",
```

Comment:
Going to assume this weird ordering was the work of EPSAT?

Reply:
yea we decided it would be easier to steal the generated FHIR messages from EPSAT rather than write the messages ourselves. shame that epsats ordering is so painful tho